RecoEgamma/EgammaTools/python/regressionModifier_cfi.py

```diff
@@ -240,6 +240,9 @@
 regressionModifier = regressionModifier94X.clone()
 
 
+from Configuration.Eras.Modifier_run2_egamma_2016_cff import run2_egamma_2016
+run2_egamma_2016.toReplaceWith(regressionModifier,regressionModifier106XUL)
+
 from Configuration.Eras.Modifier_run2_egamma_2017_cff import run2_egamma_2017
 run2_egamma_2017.toReplaceWith(regressionModifier,regressionModifier106XUL)
```

Comment:
```(run2_egamma_2016 | run2_egamma_2017 | run2_egamma_2018).toModify``` is more compact and also reasserts that the values are the same

Reply:
> (run2_egamma_2016 | run2_egamma_2017 | run2_egamma_2018).toModify is more compact and also reasserts that the values are the same


Ooh I like this syntax, I didnt know it was possible and will keep in mind for the future. I agree its much better this